testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestWriteToHiveTransactionalTableInTrino.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.hive;
+
+import io.trino.tempto.ProductTest;
+import org.testng.annotations.Test;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.tests.product.TestGroups.HMS_ONLY;
+import static io.trino.tests.product.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
+
+public class TestWriteToHiveTransactionalTableInTrino
+        extends ProductTest
+{
+    @Test(groups = {HMS_ONLY, PROFILE_SPECIFIC_TESTS})
+    public void testInsert()
```

Comment:
Can we test it for various transactional tables like we do in `testInsertIntoTransactionalTable`

Reply:
Can we split the test for unpartitioned and partitioned tables ?

Reply:
@Praveen2112 `testInsertIntoTransactionalTable` creates the ORC ACID tables though Hive which supports `insert_only` tables. Trino does not support creating `insert_only` ORC ACID tables.